fe/fe-core/src/main/java/com/starrocks/catalog/LocalTablet.java

```diff
@@ -316,6 +317,12 @@ public void getQueryableReplicas(List<Replica> allQueryableReplicas, List<Replic
         }
     }
 
+    @Override
+    public void getQueryableReplicas(List<Replica> allQueryableReplicas, List<Replica> localReplicas,
```

Comment:
‌‌Why is there no need to implement this method?

Reply:
sharenothing is not aware of warehouse. I also think the interface design is unreasonable, but currently this PR is just migration logic.